### PR TITLE
chore: implement localAuth

### LINKED ISSUE #13: Implement local authentication

### PR DESCRIPTION
**Pull request by donovanCarreno**
closes #13

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -14,6 +15,25 @@ passport.deserializeUser((id, done) => {
   done(null, id);
 });
 
+// passport-local configurtion
+passport.use(new LocalStrategy({
+    usernameField: 'email'
+  },
+  function(email, password, done) {
+    User.findOne({ email: email }, function(err, user) {
```
Comment (trevorwhealy) [inline server/server.js]: User.findOne({where: {email}}, (err, user) => {

**Comment by donovanCarreno** [comment]
fix findOne function

**Comment by cassiebaer** [comment]
:ship:

**Comment by trevorwhealy** [comment]
🚢

**Pull request merged**